etc/errors/anvil.json

```diff
@@ -964,7 +1188,7 @@
             {
               "name": "TransactionHalt",
               "code": 1,
-              "message": "Transaction {transaction_hash} execution halted, reason: {inner}",
+              "message": "Transaction {transaction_hash} execution halted:\n{inner}",
```

Comment:
Here we have `\n`, but below we don't. Probably makes sense to make things consistent.